packages/order-utils/src/order_validation_utils.ts

```diff
@@ -109,6 +109,31 @@ export class OrderValidationUtils {
             throw new Error(RevertReason.TransferFailed);
         }
     }
+    private static async _validateSignatureIsValidOrThrowAsync(
+        provider: Provider,
+        orderHash: string,
+        signedOrder: SignedOrder,
+    ): Promise<void> {
+        // Note: isValidSignatureAsync can throw if the format is incorrect
+        try {
+            const isValidSignature = await signatureUtils.isValidSignatureAsync(
+                provider,
+                orderHash,
+                signedOrder.signature,
+                signedOrder.makerAddress,
+            );
+            if (!isValidSignature) {
+                throw new Error(OrderError.InvalidSignature);
+            }
+        } catch (e) {
+            throw new Error(OrderError.InvalidSignature);
+        }
```

Comment:
This feels like something that we should fix about `isValidSignatureAsync`. Can you catch the `format is incorrect` error within it's implementation and return `false` when it happens? Otherwise all callers need to wrap it in a try/catch.